ui/src/app/course/components/editor/editor.component.html

```diff
@@ -1,44 +1,58 @@
 <div class="course-editor row">
 	<div class="col-12">{{titleEditor}}</div>
-	<form class="col-12" (submit)="onSubmit()">
+	<form class="col-12" (submit)="onSubmit()" [formGroup]="formGroup">
 		<app-loader *ngIf="isLoading"></app-loader>
 		<div class="row" *ngIf="!isLoading">
-			<div class="col-xs-12 col-sm-6">
+			<div class="col-12">
 				<div class="form-group">
 					<label class="" for="name">title</label>
-					<input class="form-control" [formControl]="course.name" type="text" id="name" name="name">
+					<input
```

Comment:
No validataion messages for these fields except Duration

Reply:
done https://github.com/taraswww777/AngularGlobalMentoringProgram2019/commit/ec6dd7dd